spec.html

```diff
@@ -48283,7 +48287,7 @@ <h1>String.prototype.substr ( _start_, _length_ )</h1>
           1. Return the substring of _S_ from _intStart_ to _intEnd_.
         </emu-alg>
         <emu-note>
-          <p>This method is intentionally generic; it does not require that its *this* value be a String object. Therefore it can be transferred to other kinds of objects for use as a method.</p>
```

Comment:
I think the "object" term is correct here - a String is a primitive, but I think the receiver is always boxed into an object before the function is evaluated.

Reply:
agree